src/components/site-header/site-header-custom-action.html.twig

```diff
@@ -0,0 +1,101 @@
+{% apply spaceless %}
```

Comment:
do we really need to have a separated twig file for the custom action?
The language switcher comes with a custom structure, so that may be justified, but here I think that it may be in the main template